events/wc_primitive_events.txt

```diff
@@ -789,3 +794,101 @@ character_event = {
 		name = EVTOPTA_WCPRM_605
 	}
 }
+}
```

Comment:
The last bracket doesn't close anything. Moreover, there are events below thus both brackets shouldn't be here.